### PR TITLE
Improve CDI alert runbooks clarity

### PR DESCRIPTION
**Pull request by arnongilboa**
**What this PR does / why we need it**:
Follow-up to #253 fixing @ousleyp comments.

**Which issue(s) this PR fixes**:
jira-ticket: https://issues.redhat.com/browse/CNV-37427

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: docs/runbooks/CDIDataImportCronOutdated.md [hunk 1]
```diff
@@ -44,14 +44,14 @@ VMs might fail to start because no boot source is available for cloning.
...
-2. Obtain the `DataImportCrons` which are not up-to-date:
+2. List the `DataImportCron` objects that are not up-to-date:
 
    ```bash
    $ kubectl get dataimportcron -A -o jsonpath='{range .items[*]}{.status.conditions[?(@.type=="UpToDate")].status}{"\t"}{.metadata.namespace}{"/"}{.metadata.name}{"\n"}{end}' | grep False
    ```
 
 3. If a default storage class is not defined on the cluster, check the
-`DataImportCron` specification for optional storage class:
+`DataImportCron` specification for `DataVolume` template storage class:
```
Comment (ousleyp) [inline docs/runbooks/CDIDataImportCronOutdated.md:54]:
```suggestion
`DataImportCron` specification for a `DataVolume` template storage class:
```

**Review by ousleyp — COMMENTED** [review]
One very small comment, but otherwise this LGTM. Thanks for making these updates!

**Review by ousleyp — APPROVED** [review]
Thanks!!

**Review by sradco — APPROVED** [review]

**Pull request merged**